src/yetibot/core/handler.clj

```diff
@@ -52,6 +54,15 @@
        ; ensure prefix is actually a command
        (filter #(command? (-> % second second second)))))
 
+(defn has-command-prefix?
+  "Returns true if body has an command matching the prefix"
```

Comment:
Sorry, i haven't noticed that the regex returned the command. I have fixed it with your suggestions. 

Thanks for the help.